ui.frontend/src/GuideBridge.js

```diff
@@ -451,6 +451,84 @@ class GuideBridge {
     getConfigsForKey(key) {
         return this.#userConfig[key] || [];
     }
+
+    /**
+     * Unloads the adaptive form, cleaning up internal state and removing DOM elements.
+     * @summary Unloads an adaptive form, removing form container view, handlers, and configurations.
+     * @param {string} [formContainerPath] - Optional path of the form container to unload. 
+     *        If not provided, unloads the currently connected form.
+     * @method
+     * @memberof GuideBridge
+     * @instance
+     * @example
+     * // Simplest - unload current connected form
+     * guideBridge.unloadAdaptiveForm();
+     * 
+     * // Unload specific form by path (for multiple forms on page)
+     * guideBridge.unloadAdaptiveForm('/content/forms/af/myform');
+     * 
+     * // Get path from form container and unload
+     * const formPath = formContainer.getPath();
+     * guideBridge.unloadAdaptiveForm(formPath);
+     * 
+     */
+    unloadAdaptiveForm(formContainerPath) {
+        // Use provided path or fall back to current formContainerPath
+        const pathToUnload = formContainerPath || this.#formContainerPath;
+        
+        if (!pathToUnload) {
+            console.warn("No form container path specified or available to unload.");
+            return;
+        }
+        
+        // Get the container element from the view
+        let container = null;
+        if (this.#formContainerViewMap[pathToUnload]) {
+            const formContainerView = this.#formContainerViewMap[pathToUnload];
+            container = formContainerView.getFormElement();
+        }
+
+        // Clean up widget elements appended to document.body
+        // These widgets are created by captcha components and datepicker that append to body
+        const widgetSelectors = [
+            '.cmp-adaptiveform-recaptcha__widget',
+            '.cmp-adaptiveform-turnstile__widget',
+            '.cmp-adaptiveform-hcaptcha__widget',
+            '.datePickerTarget'
+        ];
+        
+        widgetSelectors.forEach(selector => {
+            const widgets = document.querySelectorAll(selector);
```

Comment:
we should be selecting these inside the particular form which is being removed.

Reply:
These widgets are shared/global resources, and removing ALL of them will break other forms, its just a temporary state across all forms.

I can check if there is only one form left, when unload is called, we can clear both cache and this storage so that it is immediately garbage collected